components/plurals/src/operands.rs

```diff
@@ -15,6 +15,19 @@ use std::str::FromStr;
 ///
 /// See [full operands description](http://unicode.org/reports/tr35/tr35-numbers.html#Operands).
 ///
+/// # Data Types
+///
+/// The following types can be converted to PluralOperands:
+///
+/// - Integers, signed and unsigned
+/// - Strings representing an arbitrary-precision decimal
+/// - [`FixedDecimal`]
+///
+/// PluralRules does not support selection from a floating-point number, because floats are not
```

Comment:
I didn't intend to literally mean PluralRules the type; I more meant the concept of plural rule selection in ICU4X.  I could change it to say "This crate does not support..." instead of "PluralRules does not support..." -- thoughts?

Reply:
"This crate" works for me!